docs/contributing/contributing-icons.md

```diff
@@ -0,0 +1,120 @@
+    case 'spotify':
+      return <FaSpotify {...restProps} />
+    // ...
+  }
+}
+```
+
+3. Open `links.json` file and add a new property to the `validIcons` object. The property name must be same as the icon name (in this case, `spotify`).
+
+```json
+{
+   "validIcons": {
+      "android": "#3DDC84",
+      "spotify": "#1DB954"
+   }
+}
+```
+
+4. Open `icons.md` file and add the new icon name, usage and preview image to the icon table in alphabetical order:
+
+```markdown
+# Icons
+
+<!-- ... -->
+
+|   Socials   |     Usage     | Icons                                                                                                                                                        |    Socials     |      Usage       | Icons                                                                                                                                   |
+| :---------: | :-----------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------: | :--------------: | --------------------------------------------------------------------------------------------------------------------------------------- |
+|   Android   |   `android`   | <img src="https://user-images.githubusercontent.com/65664185/138502465-89cfadf2-6b54-4f3d-ac44-ceacdd4824ba.png" width=65% height=30%>                       |     Apple      |     `apple`      | <img src="https://user-images.githubusercontent.com/65664185/138502540-8e9b80bf-deae-4566-a41a-c63623e83c21.png" width=100% height=30%> |
+|    Book     |    `book`     | <img src="https://user-images.githubusercontent.com/76985777/145391108-f8c08f8e-679f-45a3-ad58-83ef60aa28fe.png" width=65% height=30%>                       |   Codeforces   |   `codeforces`   | <img src="https://user-images.githubusercontent.com/91655303/148160942-870fdbb4-a57c-4861-afaa-241835390645.png" width=100% height=30%> |
+
+<!-- ... -->
+
+
+|  Snapchat   |  `snapchat`   | <img src="https://user-images.githubusercontent.com/91655303/148160774-755adc38-e089-4a20-910f-292b890e2c63.png" width=65% height=30%>                       |    Spotify     |   `spotify`      | <img src="<icon_preview_url_here>" width="100%" height="30%">                                                                           |      
+
+
+<!-- ... -->
+```
+
+## Committing and pushing the changes
+
+After you've finished making changes, you need to commit and push those first. 
+If you've cloned the forked repo locally and working on it, run these commands:
+
+```
+git add src/config/links.json src/Components/Icons/GetIcons.js icons.md
+git commit -m "<commit_message>"
+git push origin main
+```
+
+This will push changes to the forked repo.
+
+Otherwise if you're working on GitHub website, first make sure that you're on the correct branch. If you don't have one, you can create it while you commit the changes.
+
+After that, do the following:
```

Comment:
I recommend highlighting this line for visibility reasons, what do you think!?

```suggestion
**After that, do the following:**
```

Reply:
Um to be honest I think it's okay without highlighting.

Reply:
Hmm, true, but in the markdown format it looks too wordy. 
Also, you know people are hard at reading docs! 😄 So I think of highlighting these to enhance the visibility.
![image](https://user-images.githubusercontent.com/88786642/195984964-895618b0-794f-427b-8486-7ca7233d2861.png)